tests/lib/server/markdown.test.ts

```diff
@@ -1,5 +1,5 @@
 // @vitest-environment happy-dom
-import { describe, expect, it } from 'vitest';
+import { describe, expect, it, vi } from 'vitest';
```

Comment:
```suggestion
import { describe, expect, it } from 'vitest';
```

I don't see any code using `vi`.